features/product_list.feature

```diff
@@ -11,10 +11,10 @@ Feature: User can see product list on the restaurant show page
     | Borat's Palace    | Östra Hmangatan 1 | Göteborg |
 
     Given the following products exist within a specific restaurant
-    | name          | restaurant      | description      | category    | price |
-    | Chicken wings | Oliver's Burger | Nice wings       | Starter     | 50 kr |
-    | Double Burger | Oliver's Burger | Tasty fat burger | Main Course | 50 kr |
-    | Apple pie     | Oliver's Burger | Sweet pie        | Dessert     | 50 kr |
+    | name          | restaurant      | description      | category    | price | image_file_link                    |
+    | Chicken wings | Oliver's Burger | Nice wings       | Starter     | 50 kr | http://www.example.com/image.jpg   |
```

Comment:
The image bg column seems to be required for the test to pass.

Reply:
It should only be required if you want to override the value set in the factory.. which isn't the case for most of those steps.
Your tests should in principle pass if you set the default value in the factory to `http://www.example.com/image.jpg` and override that when needed.